packages/hawtio/src/plugins/runtime/Metrics.tsx

```diff
@@ -0,0 +1,93 @@
+import { Card, CardBody, CardHeader, Grid, GridItem, Title } from '@patternfly/react-core'
+import React, { useEffect, useState } from 'react'
+import { runtimeService } from './runtime-service'
+import { Metric } from './types'
+import { ChartBullet } from '@patternfly/react-charts'
+
+export const Metrics: React.FunctionComponent = () => {
+  const [metrics, setMetrics] = useState<Map<string, Metric>>(new Map())
+
+  useEffect(() => {
+    const registerMetricsRequests = () => {
+      const map = new Map<string, Metric>()
```

Comment:
We don't use `Map` in general. You can use `Record` instead unless there's a specific reason to use `Map`.